lib/pages/dashboard/dashoard_page.dart

```diff
@@ -24,15 +26,26 @@ class _DashboardPageState extends State<DashboardPage> {
           Paragraph(title: "Vos clubs"),
           DashboardClubs(onClubChange: (club) => _selectCurrentClub(club)),
           Paragraph(title: "Vos équipes"),
-          if (_currentClub != null) DashboardClubTeams(club: _currentClub, onTeamFavoriteChange: (_) => _selectCurrentClub(_currentClub)),
+          if (_currentClub != null) DashboardClubTeams(
+            key: ValueKey("dashboard-club-teams-${hashList(_currentTeams)}"),
```

Comment:
Is it really required to hash the list of teams here to produce a different `ValueKey` according to teams? Can't we have just `ValueKey("dashboard-club-teams-$_currentClub")`, as this widget is specific to the currentClub teams.

If yes the you can probably also use `ObjectKey` no?

And remember that `hashCode` can have the same value for different objects.

Reply:
If I use `currentClub` for the key, when I'll remove a team of favorite teams by clicking on the `TeamCard` and then on the star, teams displayed on the dashboard will not be updated neither the agenda.

Reply:
I can use ObjectKey you're right !

Reply:
My bad, I can't because even if the `List` is modified, the address (used by the `==` operator) will still the same and the key will not change.

Reply:
Yes I've just seen in the documentation of `ObjectKey`:
> A key that takes its identity from the object used as its value

But remember that you can't use the `hashList` of the teams for the `ValueKey` as hashCode are not unique! So even if the probability is really very small, it will be very difficult to debug. `ValueKey` can have any object and it uses `==` of the objects internally to verify their equality.

Reply:
Yes but I tested with `ObjectKey` and it doesn't work.